src/kibana/plugins/dashboard/index.js

```diff
@@ -57,9 +58,17 @@ define(function (require) {
     }
   });
 
-  app.directive('dashboardApp', function (Notifier, courier, AppState, timefilter, kbnUrl) {
-    return {
-      controller: function ($scope, $route, $routeParams, $location, $http, configFile, Private, getAppState, $modal) {
+    app.directive('dashboardApp', function (Notifier, courier, AppState,
+                                            timefilter, kbnUrl, searchAuditor) {
+        return {
+          link: function(scope, elem) {
```

Comment:
maybe at  TODO here saying this is a temporary workaround until we can find exactly where in Kibana queries are submitted to the server?